docs/velox-backend-support-progress.md

```diff
@@ -20,18 +24,20 @@ The total supported functions' number for [Spark3.3 is 387](https://spark.apache
 | NS           | Not Supported. Velox backend does not support it.                                         |
 
 
+And also some notations for the function implementation's restrictions:
+
 | Value      | Description                                                                                                                                                      |
-| ---------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Mismatched | Some functions are implemented by Velox, which return results mismatched with Apache Spark. So we marked then as "Mismatched".                                   |
-| Ansi OFF   | Gluten doesn't support [ANSI mode](https://spark.apache.org/docs/latest/sql-ref-ansi-compliance.html). If it is enabled, Gluten will fall back to Vanilla Spark. |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Mismatched | Some functions are implemented by Velox, but have different semantics from Apache Spark, we mark them as "Mismatched".                                           |
```

Comment:
`Some functions are implemented by Velox for Presto, but have different semantics from Apache Spark`?

Reply:
@yma11, it is still possible that some velox functions implemented for Spark have semantic difference from Apache Spark. So I think it's good to use the PR-proposed statement.

Reply:
> it is still possible that some velox functions implemented for Spark have semantic difference from Apache Spark.

Shouldn't this be identified as bug that we should fix? We'd better not allow this case.

Reply:
For some cases, we need to fix. But for some other cases, it is not easy for us to fix. For example, in regexp functions, when to use "\\\s" to match space, vanilla Spark treats "\v" as space, but RE2 (used by velox's regexp functions) doesn't, which is a kind of semantic difference in third-party lib.